src/status_im/chat/events/send_message.cljs

```diff
@@ -19,25 +18,25 @@
 (re-frame/reg-fx
   :send-group-message
   (fn [value]
-    (protocol/send-group-message! value)))
+    #_(protocol/send-group-message! value)))
```

Comment:
Should we just remove these handlers? (`:send-group-message`,  `:send-public-group-message`, `:send-message`)